### PR TITLE
Improve my profile page

### PR DESCRIPTION
**Pull request by arothuis**
[Ticket #106202368](https://www.pivotaltracker.com/story/show/106202368)

improved my profile

### CODE REVIEW

**Comment by arothuis** [comment]
@rjkip

File: src/OpenConext/ProfileBundle/Resources/views/MyProfile/overview.html.twig [hunk 1]
```diff
@@ -6,8 +6,34 @@
...
-    <div class="table-overview-container">
-        <table class="overview-table mdl-data-table">
+    <p>{{ 'profile.my_profile.introduction'|trans }}</p>
+    <p>{{ 'profile.my_profile.explanation'|trans }}</p>
+    <p>{{ 'profile.my_profile.pi_processing'|trans }}</p>
+
+    {% if user.hasSupportContactEmail %}
+        {% set supportContactEmail = user.supportContactEmail|replace({'mailto:': ''}) %}
+
+    <p>{{ 'profile.my_profile.questions'|trans }} <a href="mailto: {{ supportContactEmail }}"><i class="prepended fa fa-envelope"></i>{{ supportContactEmail }}</a>.</p>
```
Comment (rjkip) [inline src/OpenConext/ProfileBundle/Resources/views/MyProfile/overview.html.twig]: Doesn't the space after `mailto:` break things? To be sure, I'd rather remove it :p

**Comment by arothuis** [comment]
@rjkip

**Comment by rjkip** [comment]
:ship:

**Pull request merged**